library/src/include/lib_macros.hpp

```diff
@@ -25,6 +25,17 @@
 
 #include "hipsolver.h"
 
+// Macro for Namespace
+#ifndef HIPSOLVER_BEGIN_NAMESPACE
+#define HIPSOLVER_BEGIN_NAMESPACE    \
+    namespace hipsolver {            \
+    inline namespace v2_2_0 {
```

Comment:
It occurs to me that we should either do the same thing as in rocSOLVER to keep the version number up-to-date, or add this to the bump_hipsolver_version script.

Reply:
It has been to use the version macros.